src/js/dash/segments/getSegmentListForRepresentation.js

```diff
@@ -131,7 +144,7 @@ createSegmentListFromTemplate = function(representationXml) {
                     initializationRelativeUrl = segmentTemplate.replaceIDForTemplate(initializationRelativeUrlTemplate, representationId);
 
                 initializationRelativeUrl = segmentTemplate.replaceTokenForTemplate(initializationRelativeUrl, 'Bandwidth', representationXml.getBandwidth());
-                return baseUrl + initializationRelativeUrl;
+                return [baseUrl, initializationRelativeUrl].join('/');
```

Comment:
is this known to produce good urls? Would we end up with extra slashes? Do we care about that?
Given that we use browserify, we could always require the `url` module and [resolve it](https://iojs.org/api/url.html#url_url_resolve_from_to).